gateway/platforms/war/micro/pom.xml

```diff
@@ -159,6 +159,18 @@
       <artifactId>jboss-servlet-api_3.0_spec</artifactId>
     </dependency>
     <dependency>
+      <groupId>io.swagger</groupId>
+      <artifactId>swagger-jaxrs</artifactId>
+      <optional>true</optional>
```

Comment:
Was considering making it an optional feature but changed my mind later - feel free to remove it.